Application/src/visuals/DiagramView.java

```diff
@@ -18,16 +19,42 @@
     public static ArrayList<DiagramView> list = new ArrayList<>(); // A list of all Diagram views.
```

Comment:
Could be diagramViews instead of generic "list". Also is public - static a necessity?